02-peer-protocol.md

```diff
@@ -157,12 +157,18 @@ node can offer.
 (i.e. 1/4 the more normally-used 'satoshi per 1000 vbytes') that this
 side will pay for commitment and HTLC transactions, as described in
 [BOLT #3](03-transactions.md#fee-calculation) (this can be adjusted
-later with an `update_fee` message).
+later with an `update_fee` message). Note that if
+`option_anchor_outputs` is negotiated, this `feerate_per_kw`
+can be set to the min relay fee. It is still possible to increase the fee rate
+during the lifetime of the channel using the `update_fee` message.
 
-`to_self_delay` is the number of blocks that the other node's to-self
+`to_self_delay` is the proposed number of blocks that the other node's to-self
 outputs must be delayed, using `OP_CHECKSEQUENCEVERIFY` delays; this
 is how long it will have to wait in case of breakdown before redeeming
-its own funds.
+its own funds. The non-initiator may respond with a different value in the
+`accept_message`. However, the channel will be created using the maximum of both
+proposals (referred to as `max_to_self_delay`). This allows the initiator to
+discover the other party's requirement.
```

Comment:
This seems like a pretty big departure which isn't in the scope of just the anchor output change. As is today, the asymmetry that exists between each party's preferred CSV value allow them to express their relative confidence in their ability to respond ot breach attempts in a timely manner. As an example, I may not have a tower myself, so will require the remote party to have a 900 block CSV. On the other-hand, my channel peer may have an air tight set up, and is willing to let me sweep after only 90 blocks. 

> This allows the initiator to discover the other party's requirement.

After the open+accept messages are sent, both parties are already aware of the requirements of the other.

Reply:
The argument that @rustyrussell gave is that without symmetry there is still a closing game.

Reply:
Removed to_remote csv delay from PR as discussed in yesterday's IRC meeting. We realized that in order to do it right, the remote party should also go through second level htlc transactions. This was considered too big of a change to combine with anchor outputs.